src/primitives.ts

```diff
@@ -81,6 +81,7 @@ export enum CustomUserProperties {
   DARK_MODE = 'is_dark_mode',
   EXPERT_MODE = 'is_expert_mode',
   PEER_WALLET_AGENT = 'peer_wallet_agent',
+  REFERRER = 'referrer',
```

Comment:
I'm surprised this isn't auto-collected by amplitude. Are you sure it's not just being stripped out by our proxy?

Reply:
+1 to above but if not, is this a user property? Seems like this is more related to browser / general properties. Will this be retained in some custom way or is this meant to mock the functionality of the referrer header?

Reply:
I took a look at the proxy code and did not see anywhere this is being removed. I also spent a long time trying to parse the various examples/references from amplitude and came to the conclusion that this *used to* be supported by default, but the most recent web sdk version does not automatically report this info.